source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -74,6 +74,18 @@ class ExtAuthzLoggingInfo : public Envoy::StreamInfo::FilterState::Object {
     upstream_host_ = std::move(upstream_host);
   }
 
+  bool hasFieldSupport() const { return true; }
```

Comment:
Do you really need this `hasFieldSupport` function?  It always return true, what does it do>

Reply:
it overrides the filter_state virtual method. Was missing the override identifier.